llvm/test/tools/sycl-post-link/device-globals/test_global_variable.ll

```diff
@@ -14,10 +17,15 @@ target triple = "spir64-unknown-unknown"
 %class.anon.0 = type { i8 }
 
 @_ZL7dg_int1 = internal addrspace(1) constant %"class.cl::sycl::ext::oneapi::device_global.0" zeroinitializer, align 8 #0
+; CHECK-IR: @_ZL7dg_int1 = internal addrspace(1) constant %"class.cl::sycl::ext::oneapi::device_global.0" zeroinitializer, align 8, !spirv.Decorations !0 #0
```

Comment:
I'm concerned that we are hardcoding numbers here into `!spirv.Decorations`. I suggest that we use [numeric substitution blocks](https://llvm.org/docs/CommandGuide/FileCheck.html#filecheck-numeric-substitution-blocks) instead

Reply:
Thank you for the suggestion, done!